field_manager/field_manager.F90

```diff
@@ -707,7 +705,6 @@ subroutine read_field_table_yaml(nfields, table_name)
 end do
 
 if (present(nfields)) nfields = num_fields
-call my_table%destruct
```

Comment:
Why was the table was deallocated before but not now?

Reply:
Explicit deallocation shouldn't be necessary, because neither `fmTable_t` nor any of its child structures contain any pointers. Only `allocatable` arrays, which should be deallocated automatically when they go out of scope. Therefore, because the deallocation subroutine was called immediately before `my_table` goes out of scope, it was superfluous.